lua/wikis/commons/MatchGroup/Legacy.lua

```diff
@@ -422,4 +423,127 @@ function MatchGroupLegacy:build()
 	return MatchGroup.Bracket(self.newArgs)
 end
 
+--does the same as run but returns wiki code instead of displayed bracket
+---@return string
+function MatchGroupLegacy:generate()
+	local args = self.args
+
+	self.bracketType = args.type
+	local match2mapping = self:_get(args.template, args.templateOld)
+
+	self.newArgs = {
+		args.template,
+		id = args.id,
+		store = args.store,
+		noDuplicateCheck = args.noDuplicateCheck or '{{#var:noDuplicateCheck|}}',
```

Comment:
why no dup check?

Reply:
the legacy wrapper sets it that way already
it might break stuff if we do not convert it the same way

this likely is only used in combination with transclusions

like bracket on subpage with storage and on the main page of the tournament transcluded without storage
in this case you would get the info that the id is already used even though you do not store and hence do not cause issues in lpdb

Reply:
noDuplicateCheck doesn't seem to be used outside of sc2, except as a `not store`

Reply:
```suggestion
```